mad-dog/src/tests/test_userReplicaSetNodes/verifyValidCN.js

```diff
@@ -0,0 +1,69 @@
+const _ = require('lodash')
+const ServiceCommands = require('@audius/service-commands')
+
+const { verifyUserReplicaSetStatus } = require('../test_userReplicaSetManager')
+const { monitorAllUsersSyncStatus } = require('../test_snapbackSM')
+const { getIDfromEndpoint } = require('./setNumCreatorNodes')
+
+const { getUser } = ServiceCommands
+
+const verifyCreatorNodeRemoved = async (executeAll, removedCnId, walletIndexToUserIdMap) => {
+  const userReplicaSets = await executeAll(async (libs, i) => {
+    const userId = walletIndexToUserIdMap[i]
+    let usrQueryInfo = await getUser(libs, userId)
+    // Deconstruct the comma separated value of enpdoint1,endoint2,endpoint3
+    let replicaCNIDs = usrQueryInfo.creator_node_endpoint
+      .split(",")
+      .map(getIDfromEndpoint)
+    return { userId: replicaCNIDs }
+  })
+
+  const replicaSets = userReplicaSets.reduce((acc, urs) => ({ ...acc, ...urs }), {})
+  for (let userId in replicaSets) {
+    const replicaIds = replicaSets[userId]
+    if (replicaIds.some(cnId => cnId === removedCnId)) {
+      // throw new Error(`User ID: ${userId} has replica set: ${replicaIds}, but CN ${removedCnId} should be removed`)
```

Comment:
why not throw here?

Reply:
yeah, good catch, this was because I was testing and forgot to uncomment, thx